### PR TITLE
fix restore sheet on android

### PR DESCRIPTION
**Pull request by skylarbarrera**
Before: https://cloud.skylarbarrera.com/Screenshot_20201123-104426_Rainbow-1-.jpg

After:
      w/ cloud: https://cloud.skylarbarrera.com/Screen_Recording_20201123-162933.mp4
      w/o cloud: https://cloud.skylarbarrera.com/Screen_Recording_20201123-162958.mp4

iOS: 
      w/ cloud: https://cloud.skylarbarrera.com/Screen-Shot-2020-11-23-16-19-22.85.png
      w/o cloud: https://cloud.skylarbarrera.com/Screenshot-2020-11-23-at-10.44.40-AM.jpeg

### CODE REVIEW

File: src/components/sheet/SlackSheet.android.js [hunk 1]
```diff
@@ -27,10 +27,13 @@ const Container = styled(Centered).attrs({ direction: 'column' })`
   position: absolute;
   border-top-left-radius: 20;
   border-top-right-radius: 20;
-  top: ${({ contentHeight, additionalTopPadding }) =>
-    contentHeight && additionalTopPadding
-      ? deviceUtils.dimensions.height - contentHeight
-      : 0};
+  ${({ deferredHeight, contentHeight, additionalTopPadding }) =>
```
Comment (osdnk) [inline src/components/sheet/SlackSheet.android.js:30]: can we make it less trashy?

**Review by osdnk — APPROVED** [review]
looks correct :)

File: src/components/sheet/SlackSheet.android.js (hunk 1 above)
Comment (osdnk) [reply src/components/sheet/SlackSheet.android.js:30]: extract to some function or use some meaningful names? Maybe I'm just a bit tired, but I got totally lost. 

However, at the same time I know we have already done a ton of more hacky code so idc too much

Comment (skylarbarrera) [reply src/components/sheet/SlackSheet.android.js:30]: haha I'd love to, any suggestions on how?

**Pull request merged**